packages/report-server/src/__tests__/reportBuilder/transform/parser.test.ts

```diff
@@ -48,22 +48,23 @@ describe('parser', () => {
       {
         transform: 'updateColumns',
         insert: {
-          row: '$BCD1',
-          lastAll: 'last(@all.BCD1)',
-          sumAllPrevious: 'sum(@allPrevious.BCD1)',
+          row: '= $BCD1 + $BCD2',
```

Comment:
Maybe this should be called 'arithmetic' rather than 'row'?

Reply:
Good call! I'll also add checks for all the other lookups (`@next`,`@table`,etc.)